app/dashboards/categorization_dashboard.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'administrate/base_dashboard'
+
+class CategorizationDashboard < Administrate::BaseDashboard
+  # ATTRIBUTE_TYPES
+  # a hash that describes the type of each of the model's fields.
+  #
+  # Each different type represents an Administrate::Field object,
+  # which determines how the attribute is displayed
+  # on pages throughout the dashboard.
+  ATTRIBUTE_TYPES = {
+    id: Field::Number,
+    category: Field::BelongsTo,
+    confidence: Field::Number.with_options(decimals: 2),
+    detector_version: Field::String,
+    term: Field::BelongsTo,
+    created_at: Field::DateTime,
+    updated_at: Field::DateTime
+  }.freeze
+
+  # COLLECTION_ATTRIBUTES
+  # an array of attributes that will be displayed on the model's index page.
+  #
+  # By default, it's limited to four items to reduce clutter on index pages.
+  # Feel free to add, remove, or rearrange items.
+  COLLECTION_ATTRIBUTES = %i[
+    id
+    category
+    confidence
+    detector_version
+    term
+  ].freeze
```

Comment:
I'm going to assume that we're going to be building a bespoke UI for this pretty soon, so am going to not suggest we customize this display at the moment. While the default alphabetical field order doesn't really make sense IMO, I don't think it's worth customizing at the moment if we're building a validation UI soon.

Reply:
I'm going to put in a change to this PR anyway so if you happen to have a suggestion of what to display , feel free to let me know!

Reply:
My preference for field order would probably be:
* ID (I suppose, might as well list it first)
* Term
* Category
* Confidence
* Detector Version

This would be the same for the collection, show_page, and form displays.